File: server/src/test/java/su/povolzhye/code2ai/termipus/GreeterTest.java
package su.povolzhye.code2ai.termipus;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

class GreeterTest {
  @Test
  void greet_returnsHelloMessage() {
    // Создаём экземпляр тестируемого класса Greeter
    Greeter greeter = new Greeter();
    // Вызываем метод greet с именем "Termipus"
    String result = greeter.greet("Termipus");
    // Проверяем, что результат соответствует ожидаемой строке
    assertEquals("Hello, Termipus!", result);
  }
}

File: extension/src/greeter.test.ts
import { greet } from './greeter';

test('greet returns hello message', () => {
  // Вызываем функцию greet с именем "Termipus"
  const result = greet('Termipus');
  // Ожидаем строку "Hello, Termipus!"
  expect(result).toBe('Hello, Termipus!');
});
